test/scylla-scripts-pytest/scylla_product.py

```diff
@@ -0,0 +1,2 @@
+# dummy file for testing
```

Comment:
I don't understand the purpose of this file. In what sense is it "dummy"? Where is used in the code below (I couldn't find)?

Reply:
it's a file generated by `install.sh`, I can remove the file, create a mock of it by default in the test